.github/workflows/release.yml

```diff
@@ -26,18 +26,18 @@ jobs:
     name: "Release and push to registry"
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           path: ndla/${{ github.event.repository.name }}
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           repository: NDLANO/deploy
           token: ${{ secrets.CI_GITHUB_TOKEN }}
           path: ndla/deploy
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
-      - uses: olafurpg/setup-scala@v10
+          python-version: '3.*'
```

Comment:
Når jeg testa lokalt så var det noen requirements i deploy som ikke funka i nye python 3.11, er vi sikre på at vi ikke vil fortsette å låse denne på 3.10?

Reply:
Tja. Vi kan godt låse på 3.10. Eg satte 3.10 for å få bygga i det heile tatt i går. Sikkert lurt å være sikker på at alt funker med 3.11 før vi går over.